src/cli.ts

```diff
@@ -107,6 +108,8 @@ const cli = meow(`
             --timeout to specify a global Timeout for Test Execution
             --headless currently only possible in Chrome
             --noInstall will skip the check for new Selenium Drivers (required to run offline, default: false)
+            --jUnit will create jUnit Reports (default: false)
```

Comment:
We should make sure the location where the reports are generated is specified - I'd even say that one may use `--jUnit path/to/folder` to even specify where to generate the reports? I think that would be really handy for Jenkins etc.